src/raylib/model/BoundingBox.hpp

```diff
@@ -0,0 +1,72 @@
+/*
+** EPITECH PROJECT, 2022
+** Bomberman-Global-Offensive
+** File description:
+** BoundingBox
+*/
+
+#ifndef RAYLIB_MODEL_BOUNDINGBOX_HPP_
+#define RAYLIB_MODEL_BOUNDINGBOX_HPP_
+
+extern "C"
+{
+#include <raylib.h>
+}
+
+#include "raylib/core/Vector3.hpp"
+
+namespace raylib
+{
+    namespace model
+    {
+        /// The BoundingBox Class
+        class BoundingBox {
+          public:
+            /// Deleted default constructor
+            BoundingBox() = delete;
+
+            /// BoundingBox constructor from a 3D model
+            ///
+            /// @param model the model to get the BoundingBox from
+            BoundingBox(const ::Model model);
```

Comment:
Using the `Model` from the C `Raylib` here

Reply:
need help because of cross includes otherwise